lib/actions/device.coffee

```diff
@@ -455,3 +455,55 @@ exports.init =
 				return device.uuid
 
 		.nodeify(done)
+
+exports.osUpdate =
```

Comment:
Oh no, 40 new lines of coffeescript! :-) Could we create a parallel `device_ts.ts` file, living alongside `device.coffee`, and add just this new `exports.osUpdate` to that ts file? I've done this kind of thing to other files...

Reply:
Yea sure :+1: 
I will wait for the respective SDK parts to get released and re-iterate on this a bit later (eg next Friday).